resources/assets/js/components/Like.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div>
+        <span style="margin-right:3px">{{count}}</span><i class="fa" :class="classes" @click="toggle"></i>
+    </div>
+</template>
+
+<script>
+    export default {
+        data(){
+            return {
+                count:0,
+                isLiked: false
```

Comment:
How about getting the initial status using a prop?

Reply:
Yes, i agree, i haven't fully implemented like vue component, will do today or tomorrow